### DISCUSSION TITLE
Is there a Digital Watch Face for TizenOS based watchs?

### DISCUSSION THREAD

**Discussion by philipecabrera** [trupici/G-Watch-Wear#10] [General]
I have a Gear S3 Samsung watch and would like to know if is it possible to get a full digital watch face for it? thanks for the amazing app.

**Comment by trupici** [comment]
No. There is only one face. But you can hide clock hands and display
digital time in upper half. Look for Digital panel in user manual. You can
download it from the Files section of gwatch app users group

Dňa po 17. 7. 2023, 13:18 philipecabrera ***@***.***>
napísal(a):
